src/styles/blocks/button.scss

```diff
@@ -0,0 +1,20 @@
+.button {
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  margin: 16px;
+  background: #00acdc;
+  border: 0;
+  align-items: center;
```

Comment:
The 'align-items' property is used with flex or grid containers, but '.button' doesn't seem to be displayed as a flex or grid. If you want to center text vertically, you might want to use 'display: flex;' and 'justify-content: center;' in combination with 'align-items: center;'.